product_docs/docs/biganimal/release/free_trial/index.mdx

```diff
@@ -45,45 +16,10 @@ To get started with your free trial, you'll need to [create an EDB account](deta
-
-BigAnimal offers both standard PostgreSQL and EDB Postgres Advanced Server clusters. You can try out either using the latest version or one of the previous supported versions starting at version 11. Other configuration options are limited for the free trial.
-
-You can use several equivalent ways to create a cluster:
-
-- [Using the portal](detail/create_a_cluster/create_cluster_portal)
-- [Using the CLI](detail/create_a_cluster/create_cluster_cli)
-
-## Connect 
-
-After you have a running cluster, you can connect using any PostgreSQL-compatible software. Here are a few ideas to get you going:
-
-- [Using psql](detail/connect_to_a_cluster/connect_using_psql)
-- [Using pgAdmin](detail/connect_to_a_cluster/connect_using_pgadmin)
-- [Using DBeaver](detail/connect_to_a_cluster/connect_using_dbeaver)
-
-## Experiment
-
-A database is always more interesting when it contains...data! 
-Load up some sample data and see what BigAnimal lets you do with it:
-
-- [Importing some sample data](detail/experiment/import_data)
-- [Testing backups by restoring a cluster](detail/experiment/backup_and_restore)
-- [Seeing EDB Postgres Advanced Server's Oracle compatibility](../using_cluster/06_demonstration_oracle_compatibility) in action
-
-## What you can’t do
-Since this is a free trial, there are a few nuances and limitations to what you can do with a free account:
-
-- You can't invite team members to join your cluster or assign roles in the free trial. Other team members should create their own free trial account to explore BigAnimal. 
-
-- The BigAnimal free trial interface notes the features that aren't available in the free trial. No private networking or superuser access is available in the trial. High availability, logging, monitoring, and performance benchmarking also aren't available. 
-
-## What happens when the free trial ends? 
+## Start using BigAnimal
 
-In the BigAnimal Free Trial interface, a countdown displays the time remaining for your free trial. 
+Once you have set up your account, you can log in to the [BigAnimal](https://portal.biganimal.com) portal with 90 days to use your free credits.
 
-When the free trial ends, your account continues to exist, but your cluster and its backup and data are deleted. You can't transfer your cluster from the free trial to a paid account. Your free trial account is separate from a paid BigAnimal account. 
+## What happens when you use up your free credits? 
 
-We hope two weeks is enough time to give you an idea of what BigAnimal offers, but if you're interested in _really_ seeing it in action, we also offer an evaluation of the full product, with full access to regions and configurations. If you're interested, [contact EDB sales to learn more!](https://www.enterprisedb.com/contact)
+Once your free credits have expired, if you want to continue to evaluate BigAnimal or become a paying customer, contact Sales or sign up on the [web site](https://www.enterprisedb.com/).
```

Comment:
@nataliawojcik27 is there a special URL we should point users to if we use this wording?